src/request.rs

```diff
@@ -14,6 +15,7 @@ use crate::{Data, ParseRequestError, ServerError, UploadValue, Value, Variables}
 /// variables. The names are all in `camelCase` (e.g. `operationName`).
 #[derive(Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
+#[non_exhaustive]
```

Comment:
This is actually a breaking change (semver). But `https://github.com/async-graphql/async-graphql/pull/891` has already introduced such thing....